### PR TITLE
setting orderby to none for db counts

### PR DESCRIPTION
**Pull request by ovidiul**
## Description
When doing a DB count, having an `orderby` in the query clause can be quite expensive for large datasets, so setting its value to `none` could provide some speed benefits.

### Plugin Updated: Enterprise Search

Avoid excessive resource usage by setting `orderby` to `none` for the `health validate-counts` command.

## Checklist

Please make sure the items below have been covered before requesting a review:

- [x] This change works and has been tested locally (or has an appropriate fallback).
- [x] This change works and has been tested on a Go sandbox.
- [ ] This change has relevant unit tests (if applicable).
- [ ] This change has relevant documentation additions / updates (if applicable).
- [x] I've created a changelog description that aligns with the provided examples.

## Steps to Test
1. apply the patch
2. measure time it take to run before and after the patch against

```
wp vip-search health validate-counts
```

### CODE REVIEW

**Review by sjinks — DISMISSED** [review]
LGTM. @rebeccahum could you please also take a look?

File: search/includes/classes/class-health.php [hunk 1]
```diff
@@ -101,6 +101,9 @@ public function validate_index_entity_count( array $query_args, \ElasticPress\In
 		}
 
 		try {
+			// to avoid an expensive orderby query on large datasets, we can set the orderby to none here.
+			$query_args['orderby'] = 'none';
```
[the remark this answers is not in the document]
Comment (rebeccahum) [reply search/includes/classes/class-health.php:105]: Turns out `ep_indexing_advanced_pagination` is set to `true` by default in `query_db()` and it overrides it with `'orderby' => 'ID'`, so we should turn it off then.

**Review by sjinks — APPROVED** [review]

File: search/includes/classes/class-health.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (rinatkhaziev) [reply search/includes/classes/class-health.php:105]: @ovidiul just for the future, linking related PRs in PR description will help us with context and testing.

**Comment by rinatkhaziev** [comment]
This should go with https://github.com/Automattic/ElasticPress/pull/150 so I marked as blocked.

**Comment by sjinks** [comment]
v20220222.0

**Pull request merged**